ui_testing/pages/article_page.py

```diff
@@ -83,11 +104,13 @@ def get_comment(self):
     def set_comment(self, comment):
         self.base_selenium.set_text(element="article:comment", value=comment)
 
-    def filter_by_test_plan(self, filter_text):
-        self.base_selenium.LOGGER.info(' + Filter by test plan : {}'.format(filter_text))
-        self.open_filter_menu()
-        self.filter_by(filter_element='article:filter_test_plan', filter_text=filter_text)
+    def filter_article_by(self, filter_element, filter_text, field_type='text'):
```

Comment:
yes I've updated the test case related to this method, here is the output of that test case: 

```
#1 In case I create test plan with the article that I created, user could filter with test plan ... 2019-12-30 11:30:37.845 | INFO     | ui_testing.testcases.base_test:setUp:30 - Test case : test010_create_article_with_test_plan_filter_by_test_plan
	
2019-12-30 11:30:47.156 | INFO     | ui_testing.pages.login_page:login:10 - Login admin : admin.
2019-12-30 11:31:18.157 | INFO     | ui_testing.pages.base_pages:sleep_small:39 -  Small sleep.
2019-12-30 11:31:23.163 | INFO     | ui_testing.pages.article_page:create_new_article:10 -  + Create new article.
2019-12-30 11:31:31.949 | INFO     | ui_testing.pages.base_pages:save:51 - save the changes
2019-12-30 11:31:42.095 | INFO     | ui_testing.pages.article_page:create_new_article:54 -  + Article name : fb9a32cfde
2019-12-30 11:31:42.095 | INFO     | ui_testing.pages.testplans_page:get_test_plans_page:11 -  + Get test plans page.
2019-12-30 11:31:46.701 | INFO     | ui_testing.pages.base_pages:sleep_small:39 -  Small sleep.
2019-12-30 11:31:51.707 | INFO     | ui_testing.pages.testplan_page:create_new_test_plan:71 -  Create new test plan
2019-12-30 11:31:52.396 | INFO     | ui_testing.pages.base_pages:sleep_small:39 -  Small sleep.
2019-12-30 11:31:58.041 | INFO     | ui_testing.pages.testplan_page:create_new_test_plan:78 -  With Raw Material material type
2019-12-30 11:32:00.682 | INFO     | ui_testing.pages.base_pages:sleep_tiny:35 -  Tiny sleep.
2019-12-30 11:32:02.683 | INFO     | ui_testing.pages.testplan_page:create_new_test_plan:86 -  With fb9a32cfde article
2019-12-30 11:32:07.284 | INFO     | ui_testing.pages.base_pages:save:51 - save the changes
2019-12-30 11:32:17.431 | INFO     | ui_testing.pages.testplan_page:create_new_test_plan:98 -  Test plan name : 397e47e942
2019-12-30 11:32:22.700 | INFO     | ui_testing.pages.base_pages:sleep_small:39 -  Small sleep.
2019-12-30 11:32:27.703 | INFO     | ui_testing.pages.base_pages:sleep_small:39 -  Small sleep.
2019-12-30 11:32:32.707 | INFO     | ui_testing.pages.base_pages:open_filter_menu:69 -  Open Filter
2019-12-30 11:32:32.987 | INFO     | ui_testing.pages.article_page:filter_article_by:109 -  + Filter by test plan : 397e47e942
2019-12-30 11:32:41.591 | INFO     | ui_testing.pages.base_pages:sleep_tiny:35 -  Tiny sleep.
2019-12-30 11:32:43.687 | INFO     | ui_testing.testcases.basic_tests.test02_articles:test010_create_article_with_test_plan_filter_by_test_plan:329 -  + Assert user could filter with test plan.
2019-12-30 11:32:43.834 | INFO     | ui_testing.testcases.base_test:tearDown:50 - TearDown. 	
ok
```